run_galaxy_workflow.py

```diff
@@ -78,10 +78,17 @@ def get_args():
                             action='store_true',
                             default=False,
                             help="Keeps workflow created, it will be purged if not.")
+    arg_parser.add_argument('-l', '--library-name',
+                            required=False,
+                            default=None,
+                            help="Upload results to data library with the name specified."
+                            )
     arg_parser.add_argument('--publish', action='store_true',
                             default=False, help="Keep result history and make it public/accesible.")
     arg_parser.add_argument('--accessible', action='store_true',
                             default=False, help="Keep result history and make it accessible via link only.")
+    arg_parser.add_argument('--do-not-download-results', action='store_true',
```

Comment:
On hindsight, if we are giving a library name with `-l` then that should be sufficient to signal that the results shouldn't be downloaded, and we avoid the negative flag (which is sometimes hard to get). I would remove this one and only use the other one, explaining in the help that it will not trigger a result download because it is being left in the library.

Reply:
Done in [b93822b](https://github.com/ebi-gene-expression-group/galaxy-workflow-executor/pull/34/commits/b93822b162ac0cc5087310c2bc710285558b7443)